### ISSUE TITLE
(Suggestion) Add options to limit world sizes

### ISSUE THREAD

**Issue by G01dY05h1** [Wilyicaro/Legacy-Minecraft#117]
This may seem like a strange feature to want, especially as some people see it more as a downside to legacy console, but I always thought having a small world size makes it more fun since you don't have to worry not exploring the whole world. 

You can just set your house down anywhere and know roughly where it is at all times, where all the structures are. Plus, it comes with the added bonus that most biomes/structures are placed right next to each other to insure you get a taste of everything.

**Comment by Chilly-roll** [comment]
This would be a fun inclusion, though I wonder how hard it'd be to cram the increasing amount of overworld biomes into a classic size world.. those would be some very tiny biomes haha

**Comment by G01dY05h1** [comment]
> This would be a fun inclusion, though I wonder how hard it'd be to cram the increasing amount of overworld biomes into a classic size world.. those would be some very tiny biomes haha

Well, I believe legacy had a few options regarding the world size and it's biomes, maybe an extra world size option could be added for those who want a little extra room but still want the world limit

**Comment by DraconicNEO** [comment]
I agree, I think it's a neat idea to bring back the limited smaller world size, it was one of the things that made Legacy Console edition special, even though some people dislike it.

**Comment by DraconicNEO** [comment]
> This would be a fun inclusion, though I wonder how hard it'd be to cram the increasing amount of overworld biomes into a classic size world.. those would be some very tiny biomes haha

Do they really have to include all Biomes? While the old small words included all the biomes I feel like the newer versions only included most of them, and some biomes aren't in the world. Maybe we could do that for having limited world sizes and if people don't like it, you could always keep the option to do "infinite" worlds.

**Comment by Easyidle123** [comment]
This is now a part of the [Legacy World Sizes](https://github.com/Wilyicaro/Legacy-World-Sizes/) companion mod

**Comment by creeper2eletricboogaloo** [comment]
https://modrinth.com/mod/legacy-world-sizes It is a companion mod, but exactly what your looking for and made by willy
